DevExpress.Mvvm.CodeGenerators/InitializationGenerator/AttributesGenerator.cs

```diff
@@ -283,6 +287,59 @@ class GenerateCommandAttribute : Attribute {
         ///     Specifies a custom <b>Command</b> name. The default value is <b>[ActionName]Command</b>.
         /// </summary>
         public string? Name { get; set; }
+    }";
+        const string mvvmLightSourceCode = @"    /// <summary>
```

Comment:
Same code duplicates too many times. Combine framework specific code from common strings. Also C#10 supports string interpolation for constant strings.